### ISSUE TITLE
how does this work with the parse request count?

### ISSUE THREAD

**Issue by hsavit1** [kastiglione/Parse-RACExtensions#27]
just curious if this changes anything with the number of requests made to Parse?

**Comment by kastiglione** [comment]
Nope, it's just a wrapper. It's a bridge the provides a ReactiveCocoa API on top of the standard Parse API.
